### DISCUSSION TITLE
信息录入问题 导师信息无法显示空格

### DISCUSSION THREAD

**Discussion by BruceWeiii** [note286/xduts#103] [Q&A] [answered]
环境：本地TeX Live 2022和线上overleaf都出现该问题
xdupgthesis.cls 文件：
   info / supervisor* = { Yan Shi },
实际显示：
<img width="155" alt="image" src="https://user-images.githubusercontent.com/56987659/227724642-bab528c4-cda9-4e7a-912a-189d7c5e1c23.png">
而 info / supervisor* = { Yan\ Shi },
会显示：
<img width="152" alt="image" src="https://user-images.githubusercontent.com/56987659/227724714-eacc1824-43d8-4ba1-b4ee-be42e29ccd95.png">
请问该如何解决呢

**Comment by note286** [comment, marked as answer]
Windows 10，TeX Live 2022/2023均无法复现，`xduts`版本为`6.1.4.2`。

```latex
\documentclass{xdupgthesis}
\xdusetup{info/supervisor*={Yan Shi}}
\begin{document}
\end{document}
```

![image](https://user-images.githubusercontent.com/16207292/227725245-87333839-0f32-4088-84b9-603eb9763c41.png)

**Reply by BruceWeiii** [reply]
感谢！已解决。
在xdupgthesis.cls 文件中录入信息会出现上述情况，在main.tex文件中录入就显示正常。

**Reply by note286** [reply]
好家伙，我才反应过来，`.cls`不允许用户修改的。
